### DISCUSSION TITLE
localization in different light

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1081] [General]
hi

if we map an environment like side walk in day, can we localize in nigh with different light of day ?

tnx

**Comment by matlabbe** [comment]
Is it a general localization question or related to rtabmap? You may want to read this: https://arxiv.org/abs/2103.03827

**Reply by damavand1** [reply]
it's about rtabmap

**Reply by matlabbe** [reply]
So the link I posted may help you. The quick answer is yes, though not in all cases. It will be very dependent on how many lighting variations of the same place are in the map, or which visual features are used (some like SuperPoint are a lot more robust to lighting variations than classic ones in OpenCV).
